src/styles/styles.ts

```diff
@@ -13,13 +14,54 @@ EStyleSheet.build({
   $screenWidth: screenWidth
 });
 
+Colors.loadColors({
+  background: '#000000',
+  backgroundSecondary: '#18191F',
+  white: '#FFFFFF',
+  primary: '#26a0f8',
+  danger: '#FF003D'
+});
+
+Typography.loadTypographies({
+  smallest: {
+    fontSize: 0.85 * rem,
+    lineHeight: 1.38 * rem,
+    fontFamily: variables.fontFamily.sfProTextRegular
+  },
+  small: {
+    fontSize: rem,
+    lineHeight: 1.84 * rem,
+    fontFamily: variables.fontFamily.sfProTextRegular
+  },
+  medium: {
+    fontSize: 1.15 * rem,
+    lineHeight: 1.54 * rem,
+    fontFamily: variables.fontFamily.sfProTextRegular
+  },
+  larger: {
+    fontSize: 1.4 * rem,
+    lineHeight: 1.54 * rem,
+    fontFamily: variables.fontFamily.sfProDisplayRegular
+  },
+  large: {
+    fontSize: 1.55 * rem,
+    lineHeight: 2.46 * rem,
+    fontFamily: variables.fontFamily.sfProDisplayBold
+  },
+  largest: {
+    fontSize: 1.7 * rem,
+    lineHeight: 2.46 * rem,
+    fontFamily: variables.fontFamily.sfProDisplayBold
+  }
+});
+
 export const commonStyle = createStyles({
   container: {
     paddingHorizontal: variables.contentOffset
   },
   formInput: {
```

Comment:
Please try to move all component-specific styles into files where they are used

Reply:
Styles `commonStyle.form*` may be useful not only for `InputFormGroup` but also for date pickers, selects etc